eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -22,10 +24,19 @@ def create_enrollment(*args, **kwargs):
     """
     backend function to create enrollment
     """
+    program_uuid = kwargs.get('bundle_id')
+    course_id = kwargs.get('course_id')
+
+    if program_uuid:
+        return enroll_on_program(program_uuid, *args, **kwargs)
+    else:
```

Comment:
What if the bundle_id or course_id don't exist?
On the other hand this would be safer being two separate conditions, the existence of program_uuid and the existence of course_id